Version_Control_Exercise.md

```diff
@@ -0,0 +1,183 @@
+
+Once **Person 1** has created the repository, the next step is to add the other members of the group. Click on the three dots towards the right of the newly created repository page, select "Settings," then "Collaborators," and then "Add people." You will want to add them via username, assuming everyone in the group has made a GitHub account. Once done, you all have access to your **remote repository**, where you can put your exercise code that lives "in the cloud."
+
+To actually work on the code, you will need to **clone** the remote repository to create a **local repository** on your own computer.
+ 1. Open a terminal and navigate to a folder. We recommend making one called `mrover` to store all the work you will be doing with us.
+ 2. Run the following command (replace `zwgold` with the username of the person who had created the remote repository): `git clone git@github.com:zwgold/mrover-vc-exercise.git` (Note: This command will only work if you have SSH Keys set up. Please follow the steps in the first tutorial to do so!)
+ 3. Enter your username and password if prompted.
+
+This should create a new folder called `mrover-vc-exercise`. That is your **local repository**.
+
+## Making a basic change
+At this point, everyone should have a new directory for the **local repository** called `mrover-vc-exercise`. Navigate your terminals into that folder using `cd`. You can check that you are in the right place with the command `pwd`. 
+
+Everyone run `git status`. We will run this frequently, and you should make a habit of it - it's a
+good way to check the state of your local repository. You should see that you are up-to-date with
+`origin/main` (the "origin" refers to the remote repository on github).
+
+Run the command `ls`. You should see the default `README.md` file that GitHub created automatically with the project. Feel free to go ahead and open the file in your IDE. (You could also just open the
+whole folder in your IDE.)
+
+`README.md` contains a bunch of project startup suggestions from GitHub. You'll also notice that
+the raw text of the file is in a lightweight markup language called [Markdown](https://www.markdownguide.org/basic-syntax/).
+
+**Person 2** should edit their copy of the file to remove mostly everything and replace it with a simple list of your group members. For instance:
+
+    # mrover-vc-exercise
+    Our members in our group are Zachary Goldston, Cameron Tressler, and Ashwin Gupta.
+
+**Make sure to save the file with your changes. (Here and any time you make changes!)**
+
+Everyone run `git status`. **Person 2**, who had made the changes, should see there are changes to `README.md`.
+
+### Contributing New Changes
+**Person 2** should continue to lead for this section. Others should follow along.
+
+ 1. Run `git add README.md`. This **stages** the changes so they can be committed.
+ 2. Run `git status`. That file is now staged and ready to be committed.
```

Comment:
Just for clarity, I would say "This shows that the file is now staged..." to be clear that git status doesn't do anything